main.go

```diff
@@ -65,9 +68,11 @@ func main() {
 	}
 
 	if err = (&controllers.HarborClusterReconciler{
-		Client: mgr.GetClient(),
-		Log:    ctrl.Log.WithName("controllers").WithName("HarborCluster"),
-		Scheme: mgr.GetScheme(),
+		Client:        mgr.GetClient(),
+		Log:           ctrl.Log.WithName("controllers").WithName("HarborCluster"),
+		Scheme:        mgr.GetScheme(),
+		RequeueAfter:  requeueAfter,
+		ServiceGetter: &controllers.ServiceGetterImpl{},
```

Comment:
Maybe we should add `Recorder`.
```suggestion
		ServiceGetter: &controllers.ServiceGetterImpl{},
		Recorder: mgr.GetEventRecorderFor("HarborCluster-Controller"),
```

Reply:
We haven't designed the event about the HarborClusterCR, we can add it next iteration.